src/astro/sql/operators/agnostic_stats_check.py

```diff
@@ -170,8 +170,6 @@ def prepare_failed_checks_results(
 
 
 class AgnosticStatsCheck(SqlDecoratedOperator):
-    template_fields = ("table",)
```

Comment:
Why is this no longer necessary?

Reply:
I think was a bug, since we don't have a parameter with this name. this was not caught since the Operator was not running as a task in a DAG.